ansible/management-searchengine.yml

```diff
@@ -0,0 +1,298 @@
+    become: yes
+    docker_container:
+      image: "{{ searchengine_docker_image }}"
+      name: searchengine_cache
+      cleanup: True
+      #auto_remove: yes
+      command: "set_cache_folder -c /etc/searchengine/cachedata"
+      #networks:
+      #- name: searchengine-net
+      #published_ports:
+      #- "5577:5577"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
+
+  - name: configure number of cache rows for docker searchengine
+    become: yes
+    docker_container:
+      image: "{{ searchengine_docker_image }}"
+      name: searchengine_cache
+      cleanup: True
+      #auto_remove: yes
+      command: "set_cache_rows_number -n {{ cache_rows }}"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
+
+  - name: configure secret key for docker searchengine
+    become: yes
+    docker_container:
+      image: "{{ searchengine_docker_image }}"
+      name: searchengine_cache
+      cleanup: True
+      #auto_remove: yes
+      command: "set_searchengine_secret_key -s {{ searchengine_secret_key }}"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
+
+  - name: create elasticsearch all indcies for docker searchengine
+    become: yes
+    docker_container:
+      image: "{{ searchengine_docker_image }}"
+      name: searchengine_create_index
+      cleanup: True
+      #auto_remove: yes
+      command: create_index
+      networks:
+      - name: searchengine-net
+      #published_ports:
+      #- "5577:5577"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
+
+  - name: configure search engine url for search client
+    become: yes
+    docker_container:
+      image: "{{ searchengineclient_docker_image }}"
+      name: searchengineclient_search_uri
+      cleanup: True
+      #auto_remove: yes
+      command: "set_searchengine_url -u http://searchengine:5577/"# http://127.0.0.1:5556/"
+      #networks:
+      #- name: searchengine-net
+      #published_ports:
+      #- "5567:5567"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/client:/etc/searchengineclient/"
+
+  - name: configure secret key for search client
+    become: yes
+    docker_container:
+      image: "{{ searchengineclient_docker_image }}"
+      name: searchengineclient_search_uri
+      cleanup: True
+      #auto_remove: yes
+      command: "set_client_secret_key -s {{ searchengineclient_secret_key }}"
+      state: started
+      volumes:
+      - "{{ apps_folder }}/searchengine/client:/etc/searchengineclient/"
+
+  - name: configure app data folder for search client
+    become: yes
+    docker_container:
+      image: "{{ searchengineclient_docker_image }}"
+      name: searchengineclient_search_uri
+      cleanup: True
+      auto_remove: yes
```

Comment:
This suffers from the same issue as https://github.com/IDR/deployment/pull/359#issuecomment-1023199966. Can it be safely commented for now?